pagination/keysetpagination/paginator.go

```diff
@@ -41,6 +42,13 @@ func (p *Paginator) IsLast() bool {
 	return p.isLast
 }
 
+func (p *Paginator) SortDirection() string {
```

Comment:
We cannot really change the sort direction, as we would have to flip the comparison as well. Let's just support ascending for now. Other use-cases are still possible by building the query yourself form the paginator properties.